webpack/webpack5-loader-load-themed-styles/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "@microsoft/webpack5-loader-load-themed-styles",
+  "version": "2.0.17",
```

```suggestion
  "version": "0.0.0",
```

Weird to start at major version 2. We won't be keeping this in lockstep with the other one.